helm/conjur-app-deploy/charts/app-secretless-broker/secretless_config.yaml

```diff
@@ -0,0 +1,19 @@
+version: "2"
```

Comment:
Typically, I see config files like this placed in a `files/` subdirectory under the main chart directory. I don't know if that's a best practice thing or not. Another option could be to include it in the `templates/` directory, but preface the name with an underscore, e.g. `_secretless_config.yaml` (the underscore tells Helm to not treat it like an actual manifest file).

Reply:
Moved `secretless_config.yaml` to `helm/conjur-app-deploy/charts/app-secretless-broker/files/`.
The file is read using `.Files.Get`, which is restricted from reading files in a `templates/` directory.